lib/selection-count-view.coffee

```diff
@@ -1,15 +1,28 @@
+_ = require 'underscore-plus'
+
 class SelectionCountView extends HTMLElement
+  toolTipDisposable: null
```

Comment:
Yeah, that could/should be cleaned up eventually, although it isn't doing any harm, just more of a style/consistency issue.